definitions/v3/blutopia-api.yml

```diff
@@ -54,10 +54,19 @@ search:
   paths:
     # https://hdinnovations.github.io/UNIT3D-Community-Edition-Docs/api_endpoints.html
     # https://github.com/HDInnovations/UNIT3D-Community-Edition/blob/master/app/Http/Controllers/API/TorrentController.php
-    - path: "/api/torrents/filter?api_token={{ .Config.apikey }}&name={{ if .Query.IMDBID }}{{ else }}{{ .Keywords }}{{ end }}{{ if .Query.TMDBID }}&tmdbId={{ .Query.TMDBID }}{{ else }}{{ end }}{{ if .Query.IMDBIDShort }}&imdbId={{ .Query.IMDBIDShort }}{{ else }}{{ end }}{{ if  .Query.TVDBID }}&tvdbId={{ .Query.TVDBID }}{{ else }}{{ end }}&sortField={{ .Config.sort }}&sortDirection={{ .Config.type }}&perPage=100&page=1{{ range .Categories }}&categories[]={{.}}{{end}}{{ if .Config.freeleech }}&free=1{{ else }}{{ end }}"
+    - path: "/api/torrents/filter?sortField={{ .Config.sort }}&sortDirection={{ .Config.type }}&perPage=100&page=1{{ range .Categories }}&categories[]={{.}}{{end}}{{ if .Config.freeleech }}&free=1{{ else }}{{ end }}"
```

Comment:
I'd move everything here to `inputs` and just keep the path as `/api/torrents/filter`, rather than mixing the two.

Reply:
freeleech and categories cannot; done for the rest

Reply:
If `free` cannot be `0`:

```yml
    free: "{{ if .Config.freeleech }}1{{ else }}0{{ end }}"
```

or blank:

```yml
    free: "{{ if .Config.freeleech }}1{{ else }}{{ end }}"
```

then it and `categories` can be added to inputs as:

```yml
    $raw: "{{ range .Categories }}categories[]={{.}}&{{ end }}{{ if .Config.freeleech }}free=1&{{ else }}{{ end }}"
```

e.g. https://github.com/Prowlarr/Indexers/blob/44978befdb1ae311c0b089ae4dae7435d92b393c/definitions/v3/torrentdb.yml#L76-L77

Reply:
Good to know for $raw, thanks!
pushed

Reply:
Lint is failing the latest push... I'm not seeing it